pkg/gardenlet/apis/config/v1alpha1/types.go

```diff
@@ -129,6 +129,9 @@ type GardenletControllerConfiguration struct {
 	// BackupEntry defines the configuration of the BackupEntry controller.
 	// +optional
 	BackupEntry *BackupEntryControllerConfiguration `json:"backupEntry"`
+	// Bastion defines the configuration of the Bastion controller.
+	// +optional
+	Bastion *BastionControllerConfiguration `json:"bastion"`
```

Comment:
```suggestion
	Bastion *BastionControllerConfiguration `json:"bastion,omitempty"`
```

(please also add it to `BackupBucket` and `BackupEntry`, it probably was forgotten